src/Components/NotFound.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import NotFound from './NotFound';
+
+it('Renders the page correctly', () => {
+    const { getByText, getByRole } = render(<NotFound />);
+
+    const mainHeading = getByText('Page Not Found');
+    const alertIcon = getByRole('ROLE', {name: "NAME"});
```

Comment:
When I run it on my laptop, it tells me there is no element with role ROLE and name NAME. Please change to role=alert and name="" (it's empty as it's unnamed)